CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 - Bugfix: Fixed Twitch channel-specific filters not being applied correctly. (#4529)
 - Bugfix: Fixed emote & badge tooltips not showing up when thumbnails were hidden. (#4509)
 - Bugfix: Fixed links with invalid IPv4 addresses being parsed. (#4576)
+- Bugfix: Fixed disabled items in context-menus having a weird or the default text color. (#4423)
```

Comment:
```suggestion
- Bugfix: Fixed disabled items in context-menus having a weird text-effect or the default text color. (#4423)
```
Not sure if that suggestion would be correct, but I wanted to point out that it seems to me that there is a word missing

Reply:
When I think about it, it might even have been correct, if so, sorry